auth-manager/src/main/java/com/qpay/authmanager/service/user/impl/UserServiceImpl.java

```diff
@@ -21,20 +23,24 @@ public class UserServiceImpl implements UserService, UserDetailsService {
 
     private final UserRepository userRepository;
```

Comment:
You store user in user manager, here you store only data for authentication. -> User credentials repository